install/fo-install-pythondeps

```diff
@@ -162,6 +162,8 @@ if [[ $RUNTIME ]]; then
     else
       # Only non-experimental dependencies
       su $TARGETUSER -c 'PYTHONPATH="$HOME/pythondeps" python3 -m pip install --target=$HOME/pythondeps --no-input --upgrade scancode-toolkit'
+      su $TARGETUSER -c 'PYTHONPATH="$HOME/pythondeps" python3 -m pip install --target=$HOME/pythondeps --no-input --upgrade scanoss'
+      
```

Comment:
```suggestion
      su $TARGETUSER -c 'PYTHONPATH="$HOME/pythondeps" python3 -m pip install --target=$HOME/pythondeps --no-input --upgrade scancode-toolkit scanoss'
      
```

Installing `scanoss` with `scancode-toolkit` in same step is required as the packages are installed in a custom directory with `--upgrade` flag and it replaces the folder contents :-)